aio.api.github/aio/api/github/utils.py

```diff
@@ -0,0 +1,14 @@
+
+from datetime import datetime
```

Comment:
This can be moved to the common utils where other utilities based on date time are already present?

Reply:
i remembered why it didnt do this already

as this package is in the `aio` namespace it shouldnt depend on any of the `envoy` packages

there is currently an `aio.functional` package that we could repurpose to `aio.utils` or somesuch but for now i think better to just add these here